calculator.c

```diff
@@ -0,0 +1,141 @@
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+int numberStack[50];
+int nSTop = -1;
+int operatorStack[50];
+int opSTop = -1;
+
+int evaluateExp(char exp[]);
+void calculate();
+int operatorPrecedence(char operator);
+int custom_isdigit(char ch);
+int custom_isspace(char ch);
+
+int main()
+{
+    char exp[80];
```

Comment:
Was it specified in the assignment that the string input will have max length of 80?

Reply:
No it is not specified in the assignment. I will think how can i handle it.